packages/fether-react/src/stores/healthStore.js

```diff
@@ -22,19 +22,18 @@ export const STATUS = {
   CLOCKNOTSYNC: 'CLOCKNOTSYNC', // Local clock is not sync
   DOWNLOADING: 'DOWNLOADING', // Currently downloading Parity
   GOOD: 'GOOD', // Everything's fine
-  NOINTERNET: 'NOINTERNET', // No network connection
-  OTHER: 'OTHER', // Unknown state, might have a payload
+  NOPEERS: 'NOPEERS', // Not connected to any peers
```

Comment:
Just had a small thought: these could be Symbols instead of strings.

Really not urgent though.

Reply:
good idea, I'll do this